addons/hw_drivers/tools/helpers.py

```diff
@@ -36,6 +37,22 @@
 # Helper
 #----------------------------------------------------------
 
+#----------------------------------------------------------
+# Name for the keys in odoo.conf
+#----------------------------------------------------------
+conf_keys = {
```

Comment:
wouldn't it be better to use an `Enum` than a dictionary ?
https://docs.python.org/3/library/enum.html
It is more logical, flexible and IDE can auto-complete base on the Enum constants (compared to dict where we can easily make a typo that will end up in a key_error)

Reply:
I had some reflexions about this and indeed, it's better because
- Immutability and typo checked at compile time
- No more ugly string based key (like: conf_keys['CONF_IOT_ODOO_DB_UUID'])
- I'll do the modification later
Tks